examples/generate-python-complete-models/__snapshots__/index.spec.ts.snap

```diff
@@ -7,23 +7,23 @@ Array [
 class ObjProperty: 
   def __init__(self, input):
     if hasattr(input, 'number'):
-    	self._number = input.number
+      self._number = input.number
     if hasattr(input, 'additionalProperties'):
-    	self._additionalProperties = input.additionalProperties
-
-  @property
-  def number(self):
-  	return self._number
-  @number.setter
-  def number(self, number):
-  	self._number = number
-
-  @property
-  def additionalProperties(self):
-  	return self._additionalProperties
-  @additionalProperties.setter
-  def additionalProperties(self, additionalProperties):
-  	self._additionalProperties = additionalProperties
+      self._additionalProperties = input.additionalProperties
+
+    @property
+    def number(self):
+      return self._number
+    @number.setter
+    def number(self, number):
+      self._number = number
+
+    @property
+    def additionalProperties(self):
+      return self._additionalProperties
+    @additionalProperties.setter
+    def additionalProperties(self, additionalProperties):
+      self._additionalProperties = additionalProperties
```

Comment:
This does not look right - you want to define a property `number` inside the `ObjProperty` class, right? If so, it needs to be at the same identation as the `def __init__` is, otherwise it would be considered as part of the constructor (nested functions inside `__init__()`).